content/fair/findability/findability.md

```diff
@@ -0,0 +1,65 @@
+
+Usegalaxy.eu has been developed as a European Galaxy project by multiple ELIXIR partners as well as EOSC projects and individuals from all over Europe. It is positioned as a flagship project of the German Network of Bioinformatics infrastructure (de.NBI) and part of multiple national projects. It is one of the biggest ELIXIR services and the preferred gateway for scientific computing of multiple European Open Science Cloud (EOSC) projects. Usegalaxy.eu focuses on providing a platform for analysing and sharing scientific data rather than developing standards. In other words, the European Galaxy project progressively integrates concepts, metadatas, standardisations, tools, APIs and concepts developed by related initiatives in order to bring them to application in users’ analyses and increase the overall level of practised FAIRness.
+
+Galaxy, as an open-source Virtual Research Environment (VRE), has implemented several features to align with the Findability aspect of the FAIR principles. Below is an overview of how Galaxy addresses this critical principle.
+
+Base URLs (home pages): https://usegalaxy.org, https://usegalaxy.eu, https://usegalaxy.org.au
+
+Introductory page URL: https://galaxyproject.org
+
+### Persistent Identifiers (PIDs) for Tools and Workflows
+
+Galaxy supports the assignment of Persistent Identifiers (PIDs) to tools, workflows, and datasets. PIDs ensure that research outputs remain discoverable and accessible over time, even as technologies and platforms evolve. This practice eliminates the risk of losing resources due to broken links or outdated references, making Galaxy a reliable environment for finding scientific artifacts.
+
+### RO-Crate Integration for Rich Metadata
+
+Galaxy integrates with RO-Crate, a standardized metadata framework designed to package and describe research objects such as workflows, datasets, and softwares. RO-Crate enables the creation of structured, machine-readable metadata that provides detailed descriptions of research outputs. This metadata enhances the cataloging and retrieval of research objects, making them easier to find and understand in a broader scientific context.
+
+### Searchable Repositories
+
+Galaxy provides access to searchable repositories such as the Galaxy Toolshed, a centralized platform that indexes and hosts tools and workflows shared by the global Galaxy community. Researchers can easily browse, search, and discover tools and workflows tailored to their needs, fostering collaboration and knowledge sharing. The Galaxy Toolshed functions as an "app store" for tools, ensuring that resources are well-organized and accessible to users across disciplines.
```

Comment:
* toolshed is not for Workflows
* I would write here about file-sources, zenodo as a prominent example